extra/1-factorial.js

```diff
@@ -9,8 +9,19 @@
 */
 
```

Comment:
This is a great attempt! 👍  A couple of points
- remember to use `let` or `const` when declaring a variable, in this case for variable `i`
- When you're doing the multiplication, do you need to put `i` in square brackets? What happens if you remove the square brackets? Think about what the difference is between `i` and `[i]`
Well done for attempting this one and getting a working solution 🎉